play25-conductr-bundle-lib/src/main/scala/com/typesafe/conductr/bundlelib/play/api/LocationService.scala

```diff
@@ -0,0 +1,26 @@
+package com.typesafe.conductr.bundlelib.play.api
+
+import java.net.URI
+import javax.inject.Inject
+
+import com.typesafe.conductr.bundlelib.scala.{ AbstractLocationService, CacheLike }
+import com.typesafe.conductr.lib.play.api.{ ConnectionContext, ConnectionHandler }
+
+import scala.concurrent.Future
+import scala.language.reflectiveCalls
+
+/**
+ * LocationService used to look up services using the Typesafe ConductR Service Locator.
+ */
+class LocationService @Inject() (handler: ConnectionHandler, val cc: LocationService#CC) extends AbstractLocationService(handler) {
```

Comment:
With sub type you mean `com.typesafe.conductr.lib.play.api.ConnectionContext`? Well, if we import this one (which we already do) then we are also injecting this one.